api/src/test/java/com/epam/pipeline/controller/metadata/MetadataControllerTest.java

```diff
@@ -255,8 +255,9 @@ public void shouldUploadMetadataFromFile() throws Exception {
         verify(mockMetadataApiService).uploadMetadataFromFile(any(EntityVO.class),
                 multipartFileCaptor.capture(), eq(true));
         final MultipartFile returnedFile = multipartFileCaptor.getValue();
-        assertThat(returnedFile.getContentType()).isEqualTo(MediaType.APPLICATION_OCTET_STREAM_VALUE);
         assertThat(returnedFile.getOriginalFilename()).isEqualTo("file.txt");
+        assertThat(returnedFile.getBytes()).isEqualTo("file.txt".getBytes());
+        assertRequestFile(multipartFileCaptor, "file", MediaType.APPLICATION_OCTET_STREAM_VALUE);
```

Comment:
It looks like all all these asserts can be replaced by a single call which accepts `MultipartFile` rather than captor.

```java
assertRequestFile(captor.getValue(), fileName, fileContentInBytesOrString);
``` 

Content type will probably be the same in all the similar cases. Therefore we don't have to pass it as a parameter and just use a default value `MediaType.APPLICATION_OCTET_STREAM_VALUE`.

Also I believe there was a similar case in this or other test class in the pull request. So it would be nice to use the same  `assertRequestFile` in both cases.